sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/ColumnPruningSuite.scala

```diff
@@ -34,6 +34,7 @@ class ColumnPruningSuite extends PlanTest {
     val batches = Batch("Column pruning", FixedPoint(100),
       PushDownPredicate,
       ColumnPruning,
+      RemoveNoopOperators,
```

Comment:
Is this added to remove top `Project('b :: Nil ...)`? Without that, this test can be unchanged?

Reply:
without this, a lot more tests need to be updated...

Reply:
ah, I see. :)